src/node/crypto.ts

```diff
@@ -543,7 +543,7 @@ export let createDecipher: (() => void) | undefined = undefined;
 export let Cipher: (() => void) | undefined = undefined;
 export let Decipher: (() => void) | undefined = undefined;
 
-if (!Cloudflare.compatibilityFlags.remove_nodejs_compat_eol) {
+if (!Cloudflare.compatibilityFlags.remove_nodejs_compat_eol_v22) {
```

Comment:
Those API does not seem to be in Node 22?

```
-> % node
Welcome to Node.js v22.16.0.
Type ".help" for more information.
> c = require('crypto')
> c.createCipher 
undefined
> c.createDecipher 
undefined
> c.Cipher
undefined
> c.Decipher
undefined
```

```suggestion
if (!Cloudflare.compatibilityFlags.remove_nodejs_compat_eol_v20) {
```

Reply:
Yes, they were and they weren't. They were eol'd in 22 but the exports were never actually removed. They were actually exported *as* undefined.